.gitignore

```diff
@@ -1,3 +1,6 @@
 .project
 .idea
 node_modules
+!intl/en/
+intl/*
+
```

Comment:
But [your comment](https://github.com/strongloop/loopback-datasource-juggler/pull/1022#discussion_r72483597) in my loopback-datasource-juggler, says we can get rid of `!intl/`, no?
>Does it work without this line? !intl/

And in fact it works